modules/core/include/opencv2/core/vsx_utils.hpp

```diff
@@ -144,10 +144,10 @@ VSX_FINLINE(rt) fnm(const rg& a, const rg& b)  \
     VSX_REDIRECT_2RG(vec_uint4,  vec_ushort8,  vec_mulo, __builtin_vec_mulo)
 
     // dword2 support arrived in ISA 2.07 and GCC 8+
-    VSX_IMPL_2VRG(vec_dword2,  vec_int4,  vmulesw, vec_mule)
-    VSX_IMPL_2VRG(vec_udword2, vec_uint4, vmuleuw, vec_mule)
-    VSX_IMPL_2VRG(vec_dword2,  vec_int4,  vmulosw, vec_mulo)
-    VSX_IMPL_2VRG(vec_udword2, vec_uint4, vmulouw, vec_mulo)
+    VSX_IMPL_2VRG(vec_dword2,  vec_int4,  vmulosw, vec_mule)
```

Comment:
Any reason why you changed the vec_mule (even) to use vmulosw (odd) and vice versa?

Reply:
@ChipKerchner Some operations are indirectly endian sensitive on PPC. The vector intrinsics try to hide this as best as possible.

@seiko2plus thank you for fixing.

Reply:
theses instructions sets are effected by endian mode and due to historical reasons the mnemonics are based on big-endian. in other words `vmulosw` in little-endian is returning the product of even elements.